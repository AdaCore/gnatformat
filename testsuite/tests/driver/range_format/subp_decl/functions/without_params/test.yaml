description: SubpDecl formatting for line selection.
driver: gnatformat
args:
  [
    "test.ads",
    "--range-format",
    "-SL 28",
    "-EL 29",
    "--pipe",
  ]
