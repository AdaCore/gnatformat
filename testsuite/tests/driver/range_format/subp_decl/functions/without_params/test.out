test.ads(<SubpDecl ["Random"] test.ads:28:4-28:39>) - 28:1-28:39
^
   function Random return Probability;$

